integr8ly/installation
# add uninstall playbook

add uninstall playbook
few changes to fix bugs in the install playbook
renamed `install-all.yml` to `install.yml`
updated the readme.

Test:
1. run `ansible-playbook playbooks/install.yml -e eval_github_client_id=<client> -e eval_github_client_secret=<secret>`
2. run `ansible-playbook playbooks/uninstall.yml`
3. run `ansible-playbook playbooks/install.yml -e eval_github_client_id=<client> -e eval_github_client_secret=<secret>`

There should be no failed steps.
